scripts/rollup/utils.js

```diff
@@ -1,60 +1,89 @@
 'use strict';
 
 const ncp = require('ncp').ncp;
-const join = require('path').join;
-const resolve = require('path').resolve;
+const path = require('path');
+const mkdirp = require('mkdirp');
+const rimraf = require('rimraf');
 const exec = require('child_process').exec;
 const targz = require('targz');
 
 function asyncCopyTo(from, to) {
-  return new Promise(_resolve => {
-    ncp(from, to, error => {
-      if (error) {
-        console.error(error);
-        process.exit(1);
-      }
-      _resolve();
-    });
-  });
-}
-
-function resolvePath(path) {
-  if (path[0] === '~') {
-    return join(process.env.HOME, path.slice(1));
-  } else {
-    return resolve(path);
-  }
+  return asyncMkDirP(path.dirname(to)).then(
+    () =>
+      new Promise((resolve, reject) => {
```

Comment:
Possible follow-up: We could remove this `new Promise` boilerplate here and below using [`Promise.denodeify`](https://www.npmjs.com/package/promise#promisedenodeifyfn)